src/Oceananigans.jl

```diff
@@ -11,6 +11,7 @@ export
     # Grids
     Center, Face,
     Periodic, Bounded, Flat,
+    MutableVerticalCoordinate,

```

Comment:
may be better not to export this _immediately_, instead we can have a phase where it is experimental?